src/$Handles.js

```diff
@@ -83,12 +83,17 @@ function $Handles($container, element, options){
 				}else{
 					height = ~~(rect.height);
 				}
+
+				var new_width = magnification * width;
+				var new_height = magnification * height;
+				$canvas.trigger("user-resizing", [new_width, new_height]);
```

Comment:
We definitely don't want to multiply by the magnification here. Just use `width` and `height`, those are the dimensions in document pixels that the user cares about.